web-server/v0.4/src/models/dashboard.js

```diff
@@ -55,60 +55,68 @@ export default {
       });
     },
     *fetchResults({ payload }, { call, put }) {
-      let response = yield call(queryResults, payload);
-      let results = [];
+      const response = yield call(queryResults, payload);
+      const results = [];
       let hits;
       try {
         hits = response.hits.hits;
       } catch (error) {
-        console.log('Unsuccessful run data query, no hits on matching documents: ' + error);
+        console.log(`Unsuccessful run data query, no hits on matching documents: ${error}`);
         hits = [];
       }
-      hits.map(result => {
-        let name, controller, id, start, end;
+      hits.map((result, index) => {
+        let name;
+        let controller;
+        let id;
+        let start;
+        let end;
         try {
           name = result.fields['run.name'][0];
           controller = result.fields['run.controller'][0];
           id = result.fields['run.id'][0];
           start =
-            typeof result.fields['run.start'] != 'undefined'
+            typeof result.fields['run.start'] !== 'undefined'
```

Comment:
Why are we adding the extra equal sign?  Doesn't `typeof` return an actual string?  If so, aren't we already properly comparing two strings?

Reply:
The extra equal operator (`===`) provides type checking vs the regular counterpart (`==`) that provides type coercion. In general, it is good practice to use type-safe equality operators to avoid edge case scenarios such as these which evaluate to `true`: 
```
[] == ![]
8 == "08"
```